Formula/bitwise.rb

```diff
@@ -0,0 +1,23 @@
+class Bitwise < Formula
+  desc "Terminal based bit manipulator in ncurses"
+  homepage "https://github.com/mellowcandle/bitwise"
+  url "https://github.com/mellowcandle/bitwise/releases/download/v0.32/bitwise-v0.32.tar.gz"
+  sha256 "9dbc6646e685083cd8b876c41e828e1a4a1a6ddcb1b9765b0ef76c7fbb97f6c6"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "readline"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/bitwise", "0x29A"
```

Comment:
Check the output with `assert_match` please

Reply:
Done! Updated PR.